WSL/enterprise-current.md

```diff
@@ -0,0 +1,50 @@
+---
+title: WSL for Enterprise
+description: Resources and instructions on how to best use the WSL in an Enterprise environment.
+keywords: BashOnWindows, bash, wsl, windows, windows subsystem for linux, windowssubsystem, ubuntu, debian, suse, windows 10, enterprise, deployment, offline, packaging, store, distribution, installation, install
+ms.date: 09/04/2018
+ms.topic: article
+ms.assetid: 7afaeacf-435a-4e58-bff0-a9f0d75b8a51
+ms.custom: seodec18
+---
+
+# Windows Subsystem for Linux for Enterprise
+
```

Comment:
It would be helpful to have an introduction that sets the stage for what "WSL for Enterprise" means... there seems to be some good context-setting here: https://devblogs.microsoft.com/commandline/using-wsl-in-an-enterprise/

Some ideas:
"Working in an enterprise environment, you may have security, logistics, or other reasons that require installing Windows Subsystem for Linux (WSL) without an internet connection. The [Microsoft Store for Business](https://docs.microsoft.com/microsoft-store/) can be used to install WSL and deploy Linux distributions across your enterprise company, even in an offline environment. Follow the guide below to learn how to:
- [Create an install image for WSL](#create-a-wsl-install-image)
- [Install a Linux distribution with the Microsoft Store for Business](#anchor-link-is-the-section-title)
- [Export your custom WSL image](#anchorlink)
- [Distribute your WSL image](#anchorlink)
- [Update and patch Linux distributions](#anchorlink)